modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1335,7 +1552,7 @@ Type::
 
 | `ignoreTags`
 | `array (string)`
-| `ignoreTags` is a list of tags to specify which metrics to ignore. Each metric is associated with a list of tags. More details in https://github.com/netobserv/network-observability-operator/tree/main/controllers/flowlogspipeline/metrics_definitions . Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`.
+| `ignoreTags` is a list of tags to specify which metrics to ignore. Each metric is associated with a list of tags. More details in https://github.com/netobserv/network-observability-operator/tree/main/controllers/flowlogspipeline/metrics_definitions . Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`, `nodes-flows`, `namespaces-flows`, `workloads-flows`. Namespace-based metrics are covered by both `workloads` and `namespaces` tags, hence it is recommended to always ignore one of them (`workloads` offering a finer granularity).
```

Comment:
did we add *flows metrics in 1.4? I thought we deferred that feature to 1.5? cc @jpinsonneau

Reply:
It is part of 1.4 yes:
```
// `ignoreTags` is a list of tags to specify which metrics to ignore. Each metric is associated with a list of tags. More details in https://github.com/netobserv/network-observability-operator/tree/main/controllers/flowlogspipeline/metrics_definitions .
	// Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`, `nodes-flows`, `namespaces-flows`, `workloads-flows`.
	// Namespace-based metrics are covered by both `workloads` and `namespaces` tags, hence it is recommended to always ignore one of them (`workloads` offering a finer granularity).
	//+kubebuilder:default:={"egress","packets","nodes-flows","namespaces-flows","workloads-flows","namespaces"}
	// +optional
	IgnoreTags []string `json:"ignoreTags"`
```

https://github.com/netobserv/network-observability-operator/blob/release-1.4/api/v1beta1/flowcollector_types.go#L350-L355

Adding `flows` to your list will ignore node / namespace / workload flows health graphs. It's just a shortcut.

Reply:
To clarify, the postponed part is only [NETOBSERV-1017](https://issues.redhat.com/browse/NETOBSERV-1017) about adding `resources` tag for pods & services metrics.
This will require [NETOBSERV-1284](https://issues.redhat.com/browse/NETOBSERV-1284) `white-list capability` first to avoid any mistakes on client side while manipulating these tags.

Reply:
oh, I completely missed those new tags while testing 1075 changes and while testing it now found a bug: https://issues.redhat.com/browse/NETOBSERV-1315

Reply:
Glad we had this thread then :laughing: I'm checking the bug right now. Thanks @memodi !